packages/validators/src/url/options.ts

```diff
@@ -28,6 +35,14 @@ export const whitelistSchema = z.object({
    * @defaultValue false
    */
   disallowHostnames: z.boolean().optional(),
```

Comment:
```suggestion
  disallowHostnames: z.boolean().optional().default(false),
```

Should we begin adding the default for this as well as per above documentation line; actually if we set default is `optional` needed anymore? Gotta switch from `z.infer` to `z.input` though as per https://www.ryanmr.com/posts/zod-optional-default.

Reply:
yup will add .default(false). Was relying on optional -> undefined to be falsy